decidim-core/app/cells/decidim/card_metadata_cell.rb

```diff
@@ -7,9 +7,10 @@ class CardMetadataCell < Decidim::ViewModel
     include Decidim::IconHelper
 
     alias resource model
+    delegate :blank, to: :items
 
     def show
-      return if items.blank?
+      return if blank?
```

Comment:
I'm not sure if the user of `delegate` here clarifies and simplifies the understanding

Reply:
You're right, it's an unnecessary code addition. Reverted